### DISCUSSION TITLE
Quick edit for entries

### DISCUSSION THREAD

**Discussion by KlingeSean** [craftcms/cms#11957] [Ideas] [answered]
Alot of times when testing I need to edit an entries value of only 1 field, but I have to open the entire entry and then edit the value n save.

I know this does sound like alot of lazyness from my part, but I would like there to be a quick edit button like when you edit an image's values or an included entry within an entry.

Because then a nice edit window slides from the right and lets you edit the entry you included, basically this but for the all entries page.

It would make a world of difference for testing in My opinion.

**Comment by brandonkelly** [comment, marked as answer]
It’s not super obvious, but this is already possible, by double-clicking on an entry’s status icon, or by selecting the entry and then choosing the “Edit entry” option in its bulk action menu.

<img width="248" alt="An expanded action menu for a selected entry, with “Edit entry” highlighted." src="https://user-images.githubusercontent.com/47792/191012587-7eaf0566-f923-429e-a10c-3529dcc3b6fd.png">
